gen3/bin/kube-setup-fluentd.sh

```diff
@@ -16,15 +16,30 @@ ctx="$(g3kubectl config current-context)"
 ctxNamespace="$(g3kubectl config view -ojson | jq -r ".contexts | map(select(.name==\"$ctx\")) | .[0] | .context.namespace")"
 # only do this if we are running in the default namespace
 if [[ "$ctxNamespace" == "default" || "$ctxNamespace" == "null" ]]; then
-  if (! g3kubectl --namespace=kube-system get daemonset fluentd > /dev/null 2>&1) || [[ "$1" == "--force" ]]; then
+  if (! g3kubectl --namespace=logging get daemonset fluentd > /dev/null 2>&1) || (! g3kubectl --namespace=kube-system get daemonset fluentd > /dev/null 2>&1)  || [[ "$1" == "--force" ]]; then
     ( # subshell
-      export KUBECTL_NAMESPACE=kube-system  
-      gen3 update_config fluentd-gen3 "${GEN3_HOME}/kube/services/fluentd/gen3.conf"
+      if (! g3kubectl get namepace logging > /dev/null 2>&1); then
+        g3kubectl apply -f "${GEN3_HOME}/kube/services/fluentd/fluentd-namespace.yaml"
+      fi
+      fluentdVersion="$(g3k_manifest_lookup '.versions["fluentd"]' "$manifestPath" |awk -F: '{print $2}')"
+      export KUBECTL_NAMESPACE=logging
+
+      if [ ${fluentdVersion} == "v1.10.2-debian-cloudwatch-1.0" ];
+      then
+        fluentdConfigmap="${GEN3_HOME}/kube/services/fluentd/gen3-1.10.2.conf"
```

Comment:
Add a comment here - something like:
```
# for legacy backward compatability
```
so it's obvious which is the old, and which is the new ...

Reply:
Comments added